fronts-client/src/services/__tests__/recipeQuery.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { updateImageScalingParams } from '../recipeQuery';
+
+describe("updateImageScalingParams", ()=>{
```

Comment:
👏 👏 !

Reply:
😁 Yeah I noticed images were being v. slow to load and wondered if I had introduced a bug - turned out they _were_ very slow! For an obvious reason.